src/DocumentEditor.ts

```diff
@@ -100,7 +100,12 @@ export class DocumentEditor implements vscode.Disposable {
     }
 
     private async udpateDocumentToNode(node: IDocumentNode, doc: vscode.TextDocument): Promise<void> {
-        const updatedDoc: {} = await node.update(JSON.parse(doc.getText()));
+        const docFromEditor = JSON.parse(doc.getText());
+        const updatedDoc: {} = await node.update(docFromEditor);
+        const output = util.getOutputChannel();
+        output.appendLine(JSON.stringify(docFromEditor));
+        output.appendLine("Document update performed successfully. Document now persisted");
```

Comment:
Fix this message to include more info about the document and cosmos db account.

Here's my suggestion:
```
`${timestamp}: Updated document "${documentId}" to collection "${accountName}/${databaseName}/${collectionName}"`
```

Reply:
Addressed this in the next iteration : 
I'm using the docLink in place of the path. That seems more intuitive. 
Mongo : Using . instead of /  - makes it easier for the user to copy and paste into a shell for them to see what's happening (this.db.collection.document) 

Makes sense?